ui/core/src/api/SifService/SifService.ts

```diff
@@ -290,11 +295,7 @@ export default function createSifService({
           gas: "300000", // TODO - see if "auto" setting
         };
 
-        const txHash = await client.signAndBroadcast([msg], fee, params.memo);
-
-        triggerUpdate();
-
-        return txHash;
+        return await client.signAndBroadcast([msg], fee, params.memo);
```

Comment:
Why the changes here? Why remove `triggerUpdate()`?

Reply:
@59023g trigger update does not have effect on this operation, and after updates it breaks tests because client is initialised differently for tests and `triggerUpdate` resets it and breaks the test. Removing `triggerUpdate` does not break test and they keep returning same results. I think it was put into these functions to try to update data after the transfer, but since we had block updates subscription (and now there is a stable polling every 2 seconds regardless of block updates) there is no need to call it there.